features/FEATURE_LWIP/lwip-interface/CONTRIBUTING.md

```diff
@@ -0,0 +1,129 @@
+# Description
+
+This document describes how to update mbed OS lwip stack. Mbed OS lwip stack is a copy of
+lwip master repository. Stack is located in `FEATURE_LWIP/lwip-interface/lwip` directory.
+
+When new releases or single commits are added from lwip master repository they need to be
```

Comment:
Please change the first sentence to "When you add new releases or single commits from the LWIP master repository, you must fetch them using the mbed OS LWIP fork repository."